src/lib389/lib389/replica.py

```diff
@@ -1084,7 +1088,22 @@ def set_max_age(self, value):
         :param value: The age with a time modifier of s, m, h, d, w.
         :type value: str
         """
+        self.replace('nsslapd-changelogmaxage', value)
 
+    def set_encrypt(self, value):
+        """Set the changelog encryption
+
+        :param value:
+        :type value: str
+        """
+        self.replace('nsslapd-changelogmaxage', value)
```

Comment:
This looks wrong? Should this be maxage here?

Reply:
Oh, yeah, copy and paste error. Nice catch!